### PR TITLE
2 create header

### PR DESCRIPTION
**Pull request by mastoeppler**

### CODE REVIEW

**Review by SebastianMaier03 — CHANGES_REQUESTED** [review]
grafik
The wrong header is implemented here.
Only use 1 header file that checks internally which style should be displayed.

File: app/components/header/HeaderMain.tsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import Image from 'next/image';  
+
+const HeaderMain: React.FC = () => {
+    return (
+        <header className="flex justify-between items-center p-4 text-white relative bg-cover" style={{ backgroundImage: 'url(/images/headerbg.svg)' }}>
+            <div className="flex items-center">
```
Comment (DorinaSobiecki) [inline app/components/header/HeaderMain.tsx]: This flex box is unnecessary, remove the div and change the justify in the div above. 
Please check the other parts for unnecessary CSS as well.

**Review by DorinaSobiecki — CHANGES_REQUESTED** [review]

File: app/components/header/HeaderSubpage.tsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import Image from 'next/image'; 
+
+const HeaderSubpage: React.FC = () => {
+    return (
+        <header className="flex justify-between items-center bg-[#000582] p-4 text-white relative">
```
Comment (DorinaSobiecki) [inline app/components/header/HeaderSubpage.tsx]: bg-[#000582] Please have a look at the CSS naming convention we have defined
Is relative needed here?

File: app/components/header/HeaderMain.tsx [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import Image from 'next/image';
+
+const HeaderMain: React.FC = () => {
+    return (
+        <header className="flex justify-start items-center p-4 text-white relative bg-cover text-2xl font-bold" style={{ backgroundImage: 'url(/images/headerbg.svg)', fontFamily: 'Agency FB' }}>
```
Comment (DorinaSobiecki) [inline app/components/header/HeaderMain.tsx]: Is relative needed here?

**Review by DorinaSobiecki — CHANGES_REQUESTED** [review]
At the moment there is a blue line at the bottom of the header, please fix that as well.
Screenshot 2024-05-14 200758

**Review by DorinaSobiecki — APPROVED** [review]
LGTM

**Pull request merged**